### ISSUE TITLE
Hyphenation parser fails on french patterns

### ISSUE THREAD

**Issue by thvitt** [DARIAH-DE/DARIAH-DKPro-Wrapper#27]
When loading the french hyphenation file, the parser fails with this stack trace:

```
Caused by: java.lang.ArrayIndexOutOfBoundsException: 339
        at net.davidashen.text.Hyphenator$Scanner.read(Hyphenator.java:448) ~[ddw-0.4.7-SNAPSHOT.jar:?]
        at net.davidashen.text.Hyphenator$Scanner.cc2pat(Hyphenator.java:477) ~[ddw-0.4.7-SNAPSHOT.jar:?]
        at net.davidashen.text.Hyphenator$Scanner.getSym(Hyphenator.java:386) ~[ddw-0.4.7-SNAPSHOT.jar:?]
        at net.davidashen.text.Hyphenator.loadTable(Hyphenator.java:56) ~[ddw-0.4.7-SNAPSHOT.jar:?]
        at de.tudarmstadt.ukp.dariah.annotator.HyphenationAnnotator.initHyphenator(HyphenationAnnotator.java:140) ~[ddw-0.4.7-SNAPSHOT.jar:?]
        at de.tudarmstadt.ukp.dariah.annotator.HyphenationAnnotator.process(HyphenationAnnotator.java:152) ~[ddw-0.4.7-SNAPSHOT.jar:?]
        at org.apache.uima.analysis_component.JCasAnnotator_ImplBase.process(JCasAnnotator_ImplBase.java:48) ~[ddw-0.4.7-SNAPSHOT.jar:?]
        at org.apache.uima.analysis_engine.impl.PrimitiveAnalysisEngine_impl.callAnalysisComponentProcess(PrimitiveAnalysisEngine_impl.java:385) ~[ddw-0.4.7-SNAPSHOT.jar:?]
        ... 8 more

```
The [relevant line](https://github.com/DARIAH-DE/DARIAH-DKPro-Wrapper/blob/master/src/main/java/net/davidashen/text/Hyphenator.java#L448) fails with `cc == 339 == 0x0153` (œ)

**Comment by thvitt** [comment]
> Certain hyphenation tables use encoding other than ISO-8859-1. To facilitate translation from that particular encoding to UCS, a list of codes and their unicode values can be passed to the hyphenator. See ruhyphal.tex, koicodes.txt for an example of a KOI8-R-encoded hyphenation table and a list of codes. [[TeXHyph-J](http://www.davidashen.net/texhyphj.html)]

HyphenAnnotator initializes this to a 256-byte 1:1 table, but we ship utf-8 encoded files, so there.

We should probably just ignore that table by default.

**Comment by thvitt** [comment]
Fixed in 817586b
